Drivers/SHT30/sht30.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file sht30.h
+ * @brief 
+ */
+#ifndef SHT30_H
+#define SHT30_H
+
+#include <stdint.h>
+
+//SHT30-DIS Humidity Sensor
+/**
+ * https://www.mouser.com/datasheet/2/682/Sensirion_Humidity_Sensors_SHT3x_Datasheet_digital-971521.pdf  --Datasheet
+ * 
+ */
+
+#define SWITCHBYTES(x)             (((x<<8) | (x>>8)) & 0xFFFF)      //Switches the first and second byte (This is required for all command values)
```

Comment:
I think we need to either move this to the .c file or get rid of it, I bet there is a macro like this already built into C. I made this a while ago and regret it lol